buildstockbatch/__version__.py

```diff
@@ -4,7 +4,7 @@
 __title__ = 'buildstockbatch'
 __description__ = 'Executing BuildStock projects on batch infrastructure.'
 __url__ = 'http://github.com/NREL/buildstockbatch'
-__version__ = '2022.12.0'
+__version__ = '2022.1.0'
```

Comment:
```suggestion
__version__ = '2023.1.0'
```

Reply:
Haha. I'll be writing 2022 until March at least.